neuro_san/client/direct_agent_session_factory.py

```diff
@@ -13,14 +13,14 @@
 
 from neuro_san.interfaces.agent_session import AgentSession
 from neuro_san.internals.interfaces.context_type_toolbox_factory import ContextTypeToolboxFactory
-from neuro_san.internals.graph.registry.agent_tool_registry import AgentToolRegistry
+from neuro_san.internals.graph.registry.agent_network import AgentNetwork
 from neuro_san.internals.interfaces.context_type_llm_factory import ContextTypeLlmFactory
 from neuro_san.internals.run_context.factory.master_toolbox_factory import MasterToolboxFactory
 from neuro_san.internals.run_context.factory.master_llm_factory import MasterLlmFactory
-from neuro_san.internals.graph.persistence.agent_tool_registry_restorer import AgentToolRegistryRestorer
+from neuro_san.internals.graph.persistence.agent_network_restorer import AgentNetworkRestorer
```

Comment:
AgentToolRegistryRestorer -> AgentNetworkRestorer ... and only yields an AgentNetwork - config w/out factory